tests/lwip/Makefile

```diff
@@ -1,4 +1,5 @@
 APPLICATION = lwip
+include ../Makefile.tests_common
```

Comment:
(in general I agree that all tests should the common Makefile, but there are always exceptions from the rule ;-))